### ISSUE TITLE
Look into Adding ARTICLE to My Code for Each Restaurants Information

### ISSUE THREAD

**Issue by jfc3** [jfc3/10kapart#84]
Need to look into adding ARTICLE to my code for each restaurants information.

**Comment by jfc3** [comment]
Last night attempted to add ARTICLE and SECTION around each restaurant individually which didn't help with the reader mode. Thinking the text wasn't long enough for it to recognize it. So going to try adding ARTICLE or SECTION around the entire restaurant section of the page to see if that helps.

**Comment by jfc3** [comment]
Need to work on and complete with issues #104 and #105.

**Comment by jfc3** [comment]
Being that this repository is the same as the [ATE HERE](https://github.com/jfc3/atehere) one only with smaller JSON file sizes to keep the total page load below 10K, I have stopped doing double the work.

If you want to see, the application working live go to [John F Croston III Ate Here](http://jfciii.com/atehere/).

There will be no further code updating at this time unless I try something crazy and don't want to mess with the live version.

Closing out all the open issues too.
